src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -427,7 +484,6 @@ public Response getSiteMembershipAPI(
 		requestUrl = env.getProperty("ctools.server.url")
 				+ "direct/membership/site/" + site_id + ".json?_sessionId="
 				+ sessionId;
-
 		try {
 			membersString = restTemplate.getForObject(requestUrl, String.class);
 		} catch (RestClientException e) {
```

Comment:
Like the use of catching a specific exception then followed by a more generic exception.